devicetypes/axis/axis-gear-v2.src/axis-gear-v2.groovy

```diff
@@ -0,0 +1,196 @@
+metadata {
+    definition (name: "AXIS Gear-V2", namespace: "axis", author: "AXIS Labs") {  
+        capability "Actuator"
+        capability "Configuration"
+        capability "Switch"
+        capability "Switch Level"
+        capability "Refresh"        
+        capability "Battery"
+        capability "HealthCheck"
```

Comment:
two words

Reply:
> The gear, gear v2, and gear v3 all seem identical to me and should at least be implemented as one single DTH with multiple fingerprints.
> 
> Alternatively, I think they would fit easily in our `zigbee-window-shade` DTH with only a few slight manufacturer-specific tweaks.
> 
> As a general comment, if you could please make all of your whitespace (spaces and tabs specifically) and formatting (spacing for if statements, function capitalization) consistent I would appreciate it.

the DTH that should be reviewed is v5.1 as Gear, GearV2 and GearV3 were older device handlers. V5.1 should work with older Gears that don't use window covering commands as well as newer versions of firmware that do.

As a result the older versions can be removed if required.

Reply:
Please remove them from the PR.